tutorial-datasets.csv

```diff
@@ -50,3 +50,4 @@ data_visualizing-misaligned-cords,single_subject/data/t1/t1_seg.nii.gz
 data_batch-processing-of-subjects,multi_subject/process_data.sh
 data_batch-processing-of-subjects,multi_subject/config.yml
 data_batch-processing-of-subjects,multi_subject/data/
+data_ms-lesion-segmentation,single_subject,single_subject/data/t2_ms/t2.nii.gz
```

Comment:
Tiny nitpick, but I think it would be good to place this line in the correct order for where these commands would be in the tutorials. (Presumably, we will eventually put these commands next to (or even within) the "lesion analysis" section, so maybe we can move it right after `data_lesion-analysis`?)